frontend/packages/knative-plugin/src/components/sink-source/SinkSourceController.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { createModalLauncher, ModalComponentProps } from '@console/internal/components/factory';
+import SinkSource from './SinkSource';
+
+type SinkSourceControllerProps = {
+  obj: K8sResourceKind;
```

Comment:
Can we be more specific on the prop name. Why not `source`? I see this pattern is copied from `TrafficSplittingController` ;/

Reply:
updated :)